WEATHERAPP/Components/SignUp/SignUp.css

```diff
@@ -0,0 +1,87 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
+   display: flex;
+   flex-direction: column;
+   justify-content: space-evenly;
+   align-items: center;
+}
+.app-header
+{
+   text-align: center;
+   font:700 72px system-ui;
+}
+.signup-form
+{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background-color: rgba(132, 95, 95, 1);
```

Comment:
Move colors to another css file and use using var here

Reply:
Resolved